app/templates/public/index.html

```diff
@@ -7,7 +7,7 @@
 <body>
 <h1>Welcome to <%= title %></h1>
 <ul>
-    <li><h3><a href="/api-explorer/">Interactive API Doc!</a></h3></li>
+    <li><h3><a href="/<%= title %>/api-explorer/">Interactive API Doc!</a></h3></li>
```

Comment:
title isn't correct, you may have meant root. In any case, lets leave api-explorer at the root, since its not apparent that there is a better option